src/platforms/android/common/proguard.mdx

```diff
@@ -1,10 +1,10 @@
 ---
 title: ProGuard
 sidebar_order: 100
-description: "Learn about using ProGuard to upload the ProGuard/R8 mappings, source bundle, and native symbols Sentry."
+description: "Learn about using ProGuard to upload the ProGuard/R8 mappings, source bundle, and native symbols to Sentry."
```

Comment:
```suggestion
description: "Learn about using the Sentry Gradle Plugin or sentry-cli for providing debug symbols to Sentry."
```

what's about this? similar to https://github.com/getsentry/sentry-docs/edit/master/src/platforms/react-native/upload-debug.mdx

Reply:
is the description shown somewhere? I don't see in the docs page actually.

Reply:
It seems like two pages with different content shouldn't have the exact same description, but if you think it's reasonably accurate, then let's go with it. The complaint about the description from the user was just about the typo. What about:
```suggestion
description: "Learn about using the Sentry Gradle Plugin or sentry-cli to upload the ProGuard/R8 mappings, source bundle, and native symbols to Sentry."
```

Reply:
> 

No, it's only in the metadata of the page.

Reply:
> is the description shown somewhere? I don't see in the docs page actually.

No, it's only in the metadata of the page.